Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:6
+
+# Screwdriver Store Version
+ARG VERSION=latest
+
+# Create our application directory
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+# Install Screwdriver Executor Queue
+RUN npm install screwdriver-executor-queue-worker@$VERSION
+WORKDIR /usr/src/app/node_modules/screwdriver-executor-queue-worker
```

Comment:
This is what docker suggest to do. They suggest using WORKDIR to change directory instead of RUN cd...
https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#workdir